pom.xml

```diff
@@ -300,6 +300,7 @@
                 </executions>
                 <configuration>
                     <mainClass>org.opentripplanner.standalone.OTPMain</mainClass>
+                    <cleanupDaemonThreads>false</cleanupDaemonThreads>
```

Comment:
Co to robi właściwie?

Reply:
usuwa jakiś błąd wyświetlający się po budowaniu grafu. Mnie się nie pytaj, ja tylko kopiuje ze stackoverflow

Reply:
https://stackoverflow.com/questions/13471519/running-daehttps://stackoverflow.com/questions/13471519/running-daemon-with-exec-maven-plugin-avoiding-illegalthreadstateexceptionmon-with-exec-maven-plugin-avoiding-illegalthreadstateexception